src/constants/BASE_URL.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
```

Comment:
Why file with ts end? You don't need to create a separate file for every constant